src/dwi/tractography/seeding/basic.h

```diff
@@ -160,6 +161,119 @@ namespace MR
 
 
 
+        class Coordinate_parser
+        { MEMALIGN(Coordinate_parser)
+
+          public:
+
+            Eigen::MatrixXf coords;
+            size_t nr;
+            size_t nc;         
+
+            Coordinate_parser (const std::string& cds_path) {
+              
+              coords = load_matrix<float> (cds_path);
+              nr = coords.rows();
+              nc = coords.cols();
+
+            }  
+
+        };
+
+
+        
+        class Coordinates_fixed : public Base, public Coordinate_parser
+        { MEMALIGN(Coordinates_fixed)
+
+          public:
+
+            Coordinates_fixed (const std::string& in, const size_t n_streamlines) :
+              Base (in, "coordinate seeding fixed", MAX_TRACKING_SEED_ATTEMPTS_FIXED), 
+              Coordinate_parser(in) {
+
+                if (nc != 3)
+                  throw Exception ("Number of columns in \"" + in + "\" must equal 3!");   
+                if (n_streamlines == 0)
+                  throw Exception ("Number of streamlines to be generated per coordinate must be a positive integer!");             
```

Comment:
This can instead be enforced at the command-line by setting the minimum permissible value of the corresponding argument to 1.

Reply:
Nevermind; it's already there.